test/integration/scheduler_perf/README.md

```diff
@@ -40,6 +40,22 @@ cd test/integration/scheduler_perf
 ```
 
 
+How to Run metrics based suite
```

Comment:
I think we should recommend to run them as follows:

```
make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -bench=BenchmarkPerfScheduling"
```

and to run a specific benchmark:

```
make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -bench=BenchmarkPerfScheduling/SchedulingBasic/5000Nodes/5000InitPods/1000PodsToSchedule"
```

and to produce a cpu profile:
```
make test-integration WHAT=./test/integration/scheduler_perf KUBE_TIMEOUT="-timeout=3600s" KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-alsologtostderr=false -logtostderr=false -run=^$$ -bench=BenchmarkPerfScheduling -cpuprofile ~/cpu-profile.out"
```

Reply:
Fair point, updated.